test/com/linkedin/drelephant/spark/heuristics/ExecutorsHeuristicTest.scala

```diff
@@ -16,8 +16,9 @@
 
 package com.linkedin.drelephant.spark.heuristics
 
-import scala.collection.JavaConverters
```

Comment:
Nit : Removed and added again

Reply:
It should have shown only new package as diff lines.